extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -24,41 +24,47 @@ tags: {
   }
   attr_lists: "common-extension-attrs"
 }
-# Temporarily disable until interaction of amp-script and signed exchanges has
-# been security reviewed.
-#tags: {  # <amp-script> (local script)
-#  html_format: AMP
-#  tag_name: "SCRIPT"
-#  spec_name: "amp-script extension local script"
-#  requires_extension: "amp-script"
-#  attrs: {
-#    name: "target"
-#    mandatory: true
-#    value: "amp-script"
-#    dispatch_key: NAME_VALUE_DISPATCH
-#  }
-#  attrs: {
-#    name: "type"
-#    mandatory: true
-#    value_casei: "text/plain"
-#  }
-#  attr_lists: "mandatory-id-attr"
-#  attr_lists: "nonce-attr"
-#  cdata: {
-#    max_bytes: 10000
-#    max_bytes_spec_url: "https://amp.dev/documentation/components/amp-script#faq"
-#    blacklisted_cdata_regex: {
-#      regex: "<!--"
-#      error_message: "html comments"
-#    }
-#  }
-#  spec_url: "https://amp.dev/documentation/components/amp-script"
-#}
+tags: {  # <amp-script> (local script)
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-script extension local script"
+  requires_extension: "amp-script"
+  attrs: {
+    name: "target"
+    mandatory: true
+    value: "amp-script"
+    dispatch_key: NAME_VALUE_DISPATCH
+  }
+  attrs: {
+    name: "type"
+    mandatory: true
+    value_casei: "text/plain"
+  }
+  attr_lists: "mandatory-id-attr"
+  attr_lists: "nonce-attr"
+  cdata: {
+    max_bytes: 10000
+    max_bytes_spec_url: "https://amp.dev/documentation/components/amp-script#faq"
+    blacklisted_cdata_regex: {
+      regex: "<!--"
+      error_message: "html comments"
+    }
+  }
+  spec_url: "https://amp.dev/documentation/components/amp-script"
+}
 tags: {  # <amp-script>
   html_format: AMP
   tag_name: "AMP-SCRIPT"
   disallowed_ancestor: "AMP-SCRIPT"
   requires_extension: "amp-script"
+  attrs: {
+    name: "max-age"
+    value_regex: "[0-9]+"
+    trigger: {
+      # max-age only applies to inline scripts.
+      also_requires_attr: "script"
+    }
+  }
   attrs: { name: "sandbox" }
   attrs: {
     name: "script"
```

Comment:
Consider using the ValueSet feature to validate that script="" points to an existing script tag on the page.
 - Add `AMP_SCRIPT_IDS` to https://github.com/ampproject/amphtml/blob/2424d4f555a102c61e42cf486fcab5f1321a4793/validator/validator.proto#L131.
 - Add `value_one_of_set: AMP_SCRIPT_IDS` here.
 - Copy the id attr spec from https://github.com/ampproject/amphtml/blob/2424d4f555a102c61e42cf486fcab5f1321a4793/validator/validator-main.protoascii#L5378 into the new script tag spec above, and add `add_value_to_set: AMP_SCRIPT_IDS` to it.